app/webpacker/styles/components/photo-quote-list.scss

```diff
@@ -0,0 +1,145 @@
+// grid layout
+ol.photo-quote-list .wrapper {
+  display: grid;
+  grid-template-rows: min-content 3em min-content min-content min-content;
+  grid-template-columns: $indent-amount 20% auto $indent-amount;
+
+  picture {
+    grid-area: 1 / 2 / span 2 / span 2;
+  }
+
+  h3 {
+    grid-area: 2 / 3 / span 2 / span 2;
+  }
+
+  p {
+    grid-area: 4 / 2 / span 1 / span 2;
+  }
+
+  strong {
+    grid-area: 5 / 2 / span 1 / span 2;
+  }
+
+  @include mq($from: tablet) {
+    grid-template-columns: $indent-amount auto 3 * $indent-amount $indent-amount auto $indent-amount;
+    grid-template-rows: auto;
+
+    picture {
+      grid-area: 1 / 2 / span 3 / span 2;
+    }
+
+    h3 {
+      grid-area: 1 / 3 / span 1 / span 4;
+    }
+
+    p {
+      grid-area: 2 / 5 / span 1 / span 1;
+    }
+
+    strong {
+      grid-area: 3 / 5 / span 1 / span 1;
+    }
+  }
+}
+
+// styles
+ol.photo-quote-list {
+  margin: 0;
+  list-style: none;
+  counter-reset: quotes;
+  padding: $indent-amount 0;
+
+  .item {
+    counter-increment: quotes;
+    position: relative;
+    margin: $indent-amount $indent-amount 2 * $indent-amount $indent-amount;
+    background-color: $grey;
+
+    &.blue {
+      &::before {
+        @extend .heading--box-black-on-blue;
+      }
+
+      h3 {
+        @extend .heading--highlight-blue;
+      }
+    }
+
+    &.pink {
+      &::before {
+        @extend .heading--box-pink;
+      }
+
+      h3 {
+        @extend .heading--highlight-pink;
+      }
+    }
+
+    &.green {
+      &::before {
+        @extend .heading--box-black-on-green;
+      }
+
+      h3 {
+        @extend .heading--highlight-green;
+      }
+    }
+
+    &.yellow {
+      &::before {
+        @extend .heading--box-yellow;
+      }
+
+      h3 {
+        @extend .heading--highlight-yellow;
+      }
+    }
+
+    &::before {
+      content: counter(quotes) '.';
+      position: absolute;
+      left: 0;
+      top: 0;
+      font-weight: bold;
+      transform: translate(-$indent-amount, -$indent-amount);
+      @extend .heading-xl;
+    }
+
+    .wrapper {
+      padding: $indent-amount 0;
+
+      picture img {
+        width: 100%;
+        max-height: 250px;
+        object-fit: cover;
+        object-position: top;
+      }
+
+      h3 {
+        max-width: 100%;
+        text-align: right;
```

Comment:
Is there a reason to right-align the h3s @ethax-ross @gazcarr ? 

I'd rather left-align them from an accessibility perspective if possible - plus I'm not sure where else we right-align anything?

Reply:
Just realised the Havas design right-aligned on mobile, but left-aligned on desktop

Can we left-align on both? Any brand issues with this @gazcarr ?

Reply:
@gemmadallmandfe @ethax-ross @gazcarr on desktop the quote components are  looking really quite left aligned (lots of white space to the right). Just wanted to check we're happy with this.